### PR TITLE
Add code generators for the MultiInputFile filter parameter

### PR DESCRIPTION
**Pull request by imikejackson**
Signed-off-by: Michael Jackson <mike.jackson@bluequartz.net>

### CODE REVIEW

**Comment by imikejackson** [comment]
Related PRs

https://github.com/BlueQuartzSoftware/DREAM3D/pull/966
https://github.com/dream3d/DREAM3DReview/pull/40
https://github.com/BlueQuartzSoftware/SIMPL/pull/415

**Comment by imikejackson** [comment]
https://github.com/BlueQuartzSoftware/DREAM3DCI/pull/53
https://github.com/BlueQuartzSoftware/DREAM3DCI/pull/52

**Review by mmarineBlueQuartz — APPROVED** [review]
It seems inconsistent for MultiInputFileWidgetCodeGenerator::New to be defined in the header when the other methods are not, but I don't see any other issues.

File: Source/DevHelper/CodeGenerators/MultiInputFileWidgetCodeGenerator.h [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  using ConstWeakPointer = std::weak_ptr<const Self>;
+  static Pointer NullPointer();
+
+  static Pointer New(const QString& humanLabel, const QString& propertyName, const QString& category, const QString& initValue)
+  {
+    Pointer sharedPtr(new MultiInputFileWidgetCodeGenerator(humanLabel, propertyName, category, initValue));
+    return sharedPtr;
+  }
+
+  virtual ~MultiInputFileWidgetCodeGenerator();
```
Comment (JDuffeyBQ) [inline Source/DevHelper/CodeGenerators/MultiInputFileWidgetCodeGenerator.h]:
```suggestion
  ~MultiInputFileWidgetCodeGenerator() override;
```

File: Source/DevHelper/CodeGenerators/MultiInputFileWidgetCodeGenerator.h [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+protected:
+  MultiInputFileWidgetCodeGenerator(const QString& humanLabel, const QString& propertyName, const QString& category, const QString& initValue);
+
+public:
+  MultiInputFileWidgetCodeGenerator(const MultiInputFileWidgetCodeGenerator&) = delete;            // Copy Constructor Not Implemented
+  MultiInputFileWidgetCodeGenerator(MultiInputFileWidgetCodeGenerator&&) = delete;                 // Move Constructor Not Implemented
+  MultiInputFileWidgetCodeGenerator& operator=(const MultiInputFileWidgetCodeGenerator&) = delete; // Copy Assignment Not Implemented
+  MultiInputFileWidgetCodeGenerator& operator=(MultiInputFileWidgetCodeGenerator&&) = delete;      // Move Assignment Not Implemented
+
+private:
```
Comment (JDuffeyBQ) [inline Source/DevHelper/CodeGenerators/MultiInputFileWidgetCodeGenerator.h]:
```suggestion
```

**Review by JDuffeyBQ — CHANGES_REQUESTED** [review]

**Pull request merged**